platform_google.py

```diff
@@ -54,29 +55,31 @@ def add_user_unredeemed_payment(purchase_token: str, tx_fields: SubscriptionPlan
         google_order_id=tx_fields.order_id,
         google_payment_token=purchase_token,
     )
+
+    platform_refund_expiry_ts_ms = tx_fields.event_ts_ms + base.MILLISECONDS_IN_DAY * 2
     
     with OpenDBAtPath(env.SESH_PRO_BACKEND_DB_PATH) as db:
         backend.add_unredeemed_payment(
             sql_conn=db.sql_conn,
             payment_tx=tx,
             plan=plan,
             expiry_unix_ts_ms=tx_fields.expiry_time.unix_milliseconds,
-            unredeemed_unix_ts_ms=purchase_unix_ts_ms,
+            unredeemed_unix_ts_ms=tx_fields.event_ts_ms,
+            platform_refund_expiry_ts_ms=platform_refund_expiry_ts_ms,
             err=err,
         )
 
 
-def add_user_grace_period_expiry(purchase_token: str, order_id: str, grace_duration: GoogleDuration, err: base.ErrorSink):
+def enable_payment_auto_renew(purchase_token: str, order_id: str, grace_duration: GoogleDuration, err: base.ErrorSink):
```

Comment:
Looks like we can merge this with disable_payment_auto_renew now by parameterising the auto renewal flag and making duration optional which will cut down on LOC. You can still have a helper function if you prefer, just make it thunk into the parameterised function or call the function with the appropriate parameters at the call site.

Reply:
have merged them into one func called by both